pyproject.toml

```diff
@@ -52,6 +52,7 @@ requires-python = ">=3.10"
 [project.urls]
 Code = "https://github.com/bjlittle/geovista"
 Discussions = "https://github.com/bjlittle/geovista/discussions"
+Documentation = "https://geovista.readthedocs.io"
```

Comment:
Will this add the "Documentation" link in the following list?

![image](https://github.com/bjlittle/geovista/assets/19527482/ef493bae-0c6d-4930-b486-8959ee3c26fb)

Reply:
Yup :+1:

Reply:
Perfect.  That will be very helpful.

Reply:
Great! Thanks for pointing it out :eyes: